lib/pharos/host/configurer.rb

```diff
@@ -95,6 +95,18 @@ def custom_docker?
         @host.custom_docker?
       end
 
+      # Return stringified json array for insecure registries
+      #
+      # @return [String]
+      def insecure_registries
```

Comment:
This needs specs... `.inspect` is so weird here 😄

Reply:
Now there's some specs. There might be better ways to do this than with `.inspect`, essentially we want to properly escape the quotes.